README.md

```diff
@@ -87,81 +131,199 @@ of options to rank (cf. below). For example, if a ranking question is configured
 relevant qualitative questions to compute the probability of that learner belonging to each of these groups (cf. below for
 technical details). Knowledge components representing specific areas of interest are associated with answer options
-belonging to quantitative questions. When a learner submits relevant answers for a specific section of the LPD, the LPD
-recalculates group membership probabilities and sends both group membership information and answer data from relevant
-answer options to the Adaptive Engine. (Qualitative questions and answer options are considered *relevant* if they are
-configured to influence group membership/influence recommendations, respectively.)
+belonging to quantitative questions. When a learner submits relevant answers for a specific section of an LPD instance,
+the LPD application recalculates group membership probabilities (only taking into account learner answers
+from the current LPD instance) and sends both group membership information and answer data from relevant answer options
+to the Adaptive Engine. (Qualitative questions and answer options are considered *relevant* if they are configured
+to influence group membership/influence recommendations, respectively.)
 
-## Authoring Content
+### Configuration Options
 
-There are currently two options for authoring content: Via the Django shell and via a custom management command
-(`load_data`). LPD does not provide a Django admin for creating LPD components (sections, questions, etc.). The
-management command expects data to load into the LPD's database to be supplied in the form of CSV files that live
-in a directory called `data`. This directory needs to be a sub-directory of the root directory of this repository. See
-`lpd/management/commands/load_data.py` for details on the expected format of these CSV files.
+- `lpd`:
+    - No default
+    - Learner Profile Dashboard that this knowledge component belongs to.
+      Optional for knowledge components that are associated with answer options
+      (since answer options are linked to specific LPD instances via the questions they belong to),
+      but should be set if this knowledge component represents a group
+      (to enable filtering of group scores by LPD instance/course run).
+- `kc_id`:
+    - No default
+    - String that LPD application and adaptive engine use to uniquely identify this knowledge component.
+- `kc_name`:
+    - No default
+    - Verbose name for this knowledge component.
+
+# Authoring Content
+
+There are currently two options for authoring content: Via the Django shell and via the Django admin.
+There is also a custom management command (`load_data`). This command expects data to load into the LPD applications's
+database to be supplied in the form of CSV files that live in a directory called `data`. The `data` directory
+needs to be a sub-directory of the root directory of this repository. See `lpd/management/commands/load_data.py`
```

Comment:
It might help if an example of the required directory structure is included here.